tools/roxvet/analyzers/validateimports/analyzer.go

```diff
@@ -86,6 +86,9 @@ var (
 				"github.com/stackrox/rox/pkg/protoutils",
 			),
 		},
+		"github.com/golang/mock": {
```

Comment:
This is deprecated in favor of `go.uber.org/mock`. The rest of StackRox uses the Uber library, but we still depended on `github.com/golang/mock` due to ClairCore. ClairCore now uses the Uber library, so we removed the old dependency. Adding this check here to ensure people use the correct library in the future